typescript/schemas/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@foxglove/schemas",
+  "version": "1.6.6",
+  "description": "Foxglove-defined message schemas for ROS, Protobuf, FlatBuffers, OMG IDL, and JSON",
+  "license": "MIT",
+  "author": {
+    "name": "Foxglove",
+    "email": "support@foxglove.dev",
+    "url": "https://foxglove.dev/"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/foxglove/foxglove-sdk.git",
+    "directory": "typescript/schemas"
+  },
+  "main": "./dist/typescript/schemas/src/index.js",
+  "exports": {
+    ".": "./dist/typescript/schemas/src/index.js",
+    "./internal": "./dist/typescript/schemas/src/internal.js",
+    "./jsonschema": "./dist/typescript/schemas/src/jsonschema.js"
+  },
+  "files": [
+    "dist"
+  ],
+  "packageManager": "yarn@4.6.0",
+  "scripts": {
+    "build": "tsc -b",
+    "clean": "rimraf dist",
+    "prepack": "yarn workspace foxglove-sdk update-generated-files && yarn build"
```

Comment:
Feels like this update-generated-files should not be called from the inner package maybe?

Reply:
Yeah probably not. It was already like this, but I can see an argument that this should have already happened, especially if we tagged a new version and CI already checked that the generated files are up to date.

I'll remove it.